docs/source/overview/options.md

```diff
@@ -113,8 +113,11 @@ case insensitive.
                 - `component_func: [str]`: The component function name will be loaded from `model_script`.
                 - `dummy_inputs_func: [str]`: The dummy input function name will be loaded from `model_script`.
 
-        - `config: [str]`: The config of the model can be provided as well. Such as `WhisperConfig`. See
-        [huggingface configurations](https://huggingface.co/docs/transformers/main_classes/configuration)
+        - `config: [dict]`: The config of the model can be provided as well. Such as `WhisperConfig`. See
```

Comment:
I didn't realize that we had a `hf_config.config` field. If this is true, I think we might have to update our `load_huggingface_model_from_task` and `get_hf_model_config` to use `hf_config.config` while calling `from_pretrained`.